lib/NfcoreTemplate.groovy

```diff
@@ -1,8 +1,49 @@
 /*
- * Functions to be run on completion of pipeline
+ * This file holds several functions used within the nf-core pipeline template.
  */
 
-class Completion {
+import org.yaml.snakeyaml.Yaml
+
+class NfcoreTemplate {
+
+    /*
+     * Check AWS Batch related parameters have been specified correctly
+     */
+    public static void awsBatch(workflow, params) {
+        if (workflow.profile.contains('awsbatch')) {
+            assert (params.awsqueue && params.awsregion) : "Specify correct --awsqueue and --awsregion parameters on AWSBatch!"
+            // Check outdir paths to be S3 buckets if running on AWSBatch
+            // related: https://github.com/nextflow-io/nextflow/issues/813
+            assert params.outdir.startsWith('s3:')       : "Outdir not on S3 - specify S3 Bucket to run on AWSBatch!"
+            // Prevent trace files to be stored on S3 since S3 does not support rolling files.
+            assert !params.tracedir.startsWith('s3:')    :  "Specify a local tracedir or run without trace! S3 cannot be used for tracefiles."
```

Comment:
Thought this was solved with `overwrite = true` as reported on [this](https://github.com/nextflow-io/nextflow/issues/813) nextflow closed issue.

Reply:
Yes, I believe you are right 👍🏽 Think we can remove that now because we have gone with using a unique filename https://github.com/nf-core/rnaseq/blob/9d05b22569fc8db0407f9e2bf03e60906048b327/nextflow.config#L204 as opposed to `overwrite = true`.

Reply:
Excellent spot! https://github.com/nf-core/rnaseq/pull/622/commits/4bf388848554b54de0fa3ea77eedea119b15d3e5